src/bcrhp/bcrhp/templates/views/report-templates/bcrhp_default.htm

```diff
@@ -57,8 +57,59 @@ <h4 class="report-toolbar-title"><span class="bc-report-title" data-bind="text:
     </div>
 </div>
 
-<div class="rp-report-container-preview">
-    <div class="rp-report-section relative rp-report-section-root">
+<div class="rp-report-container-preview relative">
+    <aside id="card-help-panel" class="card-help-panel" style="display: none;" data-bind="visible: helpenable">
+        <div class="relative">
+            <a id="add-basemap-wizard-help-close" href="#" class="help-close fa fa-times fa-lg" style="" data-bind="click: function () { helpactive(false) }"></a>
+        </div>
+        <div id="add-basemap-wizard-help-content">
```

Comment:
@jrods - Would you please structure the content with `dt` and `dd` tags (or other tags if you'd prefer) so it can be controlled using css?